src/external/lib/view-engine/filters/index.js

```diff
@@ -1,19 +1,17 @@
 module.exports = {
-  ...require('./abstraction-period'),
-  ...require('./abstraction-reform'),
-  ...require('./date'),
-  ...require('./fixed'),
-  ...require('./form'),
-  ...require('./markdown'),
-  ...require('./merge'),
-  ...require('./most-significant-entity-role'),
-  ...require('./number'),
-  ...require('./pluralize'),
-  ...require('./query-string'),
-  ...require('./slice'),
-  ...require('./sort-new-direction'),
-  ...require('./sort-query'),
-  ...require('./title-case'),
-  ...require('./unit-conversion'),
-  ...require('./units')
+  ...require('../../../../shared/view/nunjucks/filters/abstraction-period'),
```

Comment:
Would these work without the `../../../../` since we are using the `app-module-path` node module?